src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -88,27 +131,22 @@ protected static ConfigDef baseConfigDef() {
       int order = 0;
       configDef
           .define(TYPE_NAME_CONFIG, Type.STRING, Importance.HIGH,
-                  "The Elasticsearch type name to use when indexing.",
+                  TYPE_NAME_DOC,
                   group, ++order, Width.SHORT, "Type Name")
```

Comment:
Not sure it's even an option with checkstyle. tbh while I generally don't mind 1-per-line when you have to overflow to multiple lines, it can get annoying in these cases where we have a lot of calls with a lot of arguments (though this is also why people want a builder interface for these...)